cmd/gaia/app/export.go

```diff
@@ -119,7 +107,7 @@ func (app *GaiaApp) prepForZeroHeightGenesis(ctx sdk.Context) {
 	}
 
 	// reset fee pool height, save fee pool
-	feePool.TotalValAccum.UpdateHeight = 0
+	feePool.TotalValAccum = distr.TotalAccum{0, sdk.ZeroDec()}
```

Comment:
use `NewTotalAccum(0)` (need to add to alias too actually... probs ma bad)

Reply:
Yeah, there wasn't an alias - added and updated.